b02cf9e61b861d10d806dc23b24bcb3f/update/sys_script_5a6efbf22f9ad11081e256f62799b69f.xml

```diff
@@ -0,0 +1,102 @@
+    <sys_script action="INSERT_OR_UPDATE">
+        <abort_action>false</abort_action>
+        <access>package_private</access>
+        <action_delete>false</action_delete>
+        <action_insert>true</action_insert>
+        <action_query>false</action_query>
+        <action_update>false</action_update>
+        <active>true</active>
+        <add_message>false</add_message>
+        <advanced>true</advanced>
+        <change_fields>false</change_fields>
+        <client_callable>false</client_callable>
+        <collection>x_snc_slackerbot_payload</collection>
+        <condition/>
+        <description/>
+        <execute_function>false</execute_function>
+        <filter_condition/>
+        <is_rest>false</is_rest>
+        <message/>
+        <name>Parse emoji add/remove events</name>
+        <order>100</order>
+        <priority>200</priority>
+        <rest_method/>
+        <rest_method_text/>
+        <rest_service/>
+        <rest_service_text/>
+        <rest_variables/>
+        <role_conditions/>
+        <script><![CDATA[(function executeRule(current, previous /*null when async*/ ) {
+
+    var payload = JSON.parse(current.payload);
+    if (payload.event.type != 'emoji_changed') return;
+
+    var event = payload.event;
+
+    var text = "";
+    switch (event.subtype) {
+        case ("add"):
+            if (event.value.indexOf("alias:") == 0) {
+                text = "A new emoji has been *aliased*! `:" + event.name + ":` pointing to :point_right: `:" + event.value.split(":")[1] + ":`";
+            } else {
+                text = "A new emoji has been *added*! `:" + event.name + ":` :" + event.name + ":";
+                var slackMessageBlocks = [];
+                var message = {};
+                message.type = "section";
+                message.text = {
+                    "type": "mrkdwn",
+                    "text": text
+                };
+                slackMessageBlocks.push(message);
+                message = {};
+                message.type = "section";
+                message.text = {
+                    "type": "plain_text",
+                    "emoji": true,
+                    "text": ":" + event.name + ":"
+                };
+                message.accessory = {
+                    "type": "image",
+                    "image_url": event.value,
+                    "alt_text": ":" + event.name + ":",
+                };
+                slackMessageBlocks.push(message);
+				text = {
+					text: text,
+					blocks: JSON.stringify(slackMessageBlocks)
+				};
+            }
+            break;
+        case ("remove"):
+            text = "An emoji has been *removed*! `:" + event.names[0] + ":`";
+            break;
+        case ("rename"):
+            text = "An emoji has been *renamed* from `:" + event.old_name + ":` to `:" + event.new_name + ":`";
+            break;
+    }
+
+
+    var chat_gr = {
```

Comment:
This isn't a gliderecord!!! 😠

Reply:
I was trying to stick to the naming convention of the params for clarity! 👀

Reply:
😛